src/abstractdataframe/selection.jl

```diff
@@ -1,8 +1,8 @@
 # TODO:
 # * add transform and transfom! functions
-# * add `Col` wrapper for whole column operations
 # * update documentation
 # * add tests
+# * add Splat to column selector
```

Comment:
We need the contrary of `Splat` now, right?

Reply:
In the long run - yes. But we do not have to have it in this PR (we can add it any time later as it is non-breaking). The major question as usual is the name, I have earlier proposed `NT`, but it does not seem ideal.

Reply:
I just mean that `Splat` is the default now, so we don't need it, we rather need `NT` or an equivalent name.

Reply:
Yes, but do you think that `NT` is a good name? I will update the entry in the file when we settle on the name :smile:.